stories/Molecules/Blocks/DownloadCard/DownloadCard.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './DownloadCard.scss';
+import '../../../assets/scss/_grid.scss';
+import { Publicationthumb } from '../../../Atom/Cards/Publication-thumbnail/PublicationThumbnail';
+import { Cardthumb } from '../../../Atom/Cards/Card-thumbnail/CardThumbnail';
+import { Icondownload } from '../../../Atom/Global/Icons/download';
+
+export const column_options = {
+  large: 'large-8 feature__card--large',
+  medium: 'medium-6 large-4 feature__card--medium',
+  small: 'medium-5 large-3 feature__card--small test1'
+}
+
+export const variant_options = {
+  Image: 'with-image',
+  Color: 'without-image',
+}
+
+export const DownloadCard = ({ image, image2, title, format, download, ...args }) => {
+  return (
+    <div class="download__card">
+      {image && <Publicationthumb />}
+      {image2 && <Cardthumb />}
+      <div class="download__card__description">
```

Comment:
@harishkumar-srijan it should be download-card as __ will be used with the element of block

Reply:
done